core/src/main/resources/schema/app/20200824205107_CB-1068_multi_az.sql

```diff
@@ -0,0 +1,49 @@
+-- // CB-1068 multi az
+-- Migration SQL that makes the change goes here.
+
+CREATE TABLE IF NOT EXISTS instancegroupnetwork (
+    id bigint NOT NULL,
+    cloudPlatform CHARACTER VARYING(255),
+    attributes TEXT,
+    stack_network_id bigint,
+    instancegroup_id bigint,
+    subnetid TEXT,
```

Comment:
I know this is only a tmp column. `instancemetadata.subnetid` is a `VARCHAR(255)`, so perhaps the same type could be sufficient here too.

Reply:
is this matter?